public/pages/Findings/containers/Findings/Findings.tsx

```diff
@@ -73,15 +77,33 @@ interface FindingsProps extends RouteComponentProps, DataSourceProps {
   setDateTimeFilter?: Function;
 }
 
-interface FindingsState {
-  loading: boolean;
+enum FindingTabId {
```

Comment:
rename it to `FindingTab` instead?

Reply:
No, since `Tab` refers to the entire object with id, name and content. Keeping it id, since that is what it's fields are